### PR TITLE
Feature: display core L1 memory buffers

### PR DESCRIPTION
**Pull request by madcampos**
Add display of L1 memory to cores.

Screenshot of new L1 core display.

### CODE REVIEW

File: src/renderer/components/detailed-view-components/DetailedViewCore.tsx [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+
+    const binaryBuffers = getChartData(node.coreL1Memory.binaryBuffers, true);
+    const binaryBuffersConfig = { ...L1RenderConfiguration };
+    binaryBuffersConfig.title = 'Binary Buffers';
+    const binaryBufferZoomRangeStart = useMemo(
+        () => (node.coreL1Memory.binaryBuffers[0]?.address ?? 0) * MINIMAL_MEMORY_RANGE_OFFSET,
+        [node.coreL1Memory.binaryBuffers],
+    );
+
+    const binaryBufferZoomRangeEnd = useMemo(() => {
```
Comment (aidemsined) [inline src/renderer/components/detailed-view-components/DetailedViewCore.tsx:51]: does this really need a useMemo?

Comment (madcampos) [reply src/renderer/components/detailed-view-components/DetailedViewCore.tsx:51]: I think it won't be a problem as it is not a very expensive computation.
But without it, the computation would happen every time the user opens the details view and they click the zoom switch.

File: src/renderer/components/detailed-view-components/DetailedViewCore.tsx [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+interface DetailedViewCoreRendererProps {
+    node: ComputeNode;
+    temporalEpoch: number;
+}
+
+const DetailedViewCoreRenderer: React.FC<DetailedViewCoreRendererProps> = ({ node, temporalEpoch }) => {
+    const MINIMAL_MEMORY_RANGE_OFFSET = 0.998;
+    const [zoomedInView, setZoomedInView] = useState(false);
+
+    // TODO: we need to find a creative way to render CONSUMED size vs SIZE - this might become interesting metrics to show
```
Comment (aidemsined) [inline src/renderer/components/detailed-view-components/DetailedViewCore.tsx:23]: we probably dont need this todo anymore

File: src/renderer/components/detailed-view-components/DetailedViewCore.tsx (hunk 1 above)
Comment (aidemsined) [reply src/renderer/components/detailed-view-components/DetailedViewCore.tsx:51]: it a loop over 20 item list max

**Review by aidemsined — APPROVED** [review]

**Pull request merged**